test/e2e/workflow/validation_test.go

```diff
@@ -41,6 +42,7 @@ var _ = Describe("NetworkAddonsConfig", func() {
 			configSpec := opv1alpha1.NetworkAddonsConfigSpec{
 				LinuxBridge: &opv1alpha1.LinuxBridge{},
 				NMState:     &opv1alpha1.NMState{},
+				MacvtapCni:  &opv1alpha1.MacvtapCni{},
```

Comment:
No need to deploy this additionally. Point of the test is to deploy some minimal valid config.

Reply:
OK.